Project/feature-u-soundboard/src/features/common/baseUI/component/MainLayout.tsx

```diff
@@ -19,6 +21,8 @@ export default function MainLayout({ children }) {
         ]}
         title={'Soundboard'}
       />
+      <UploadForm />
+      <SoundContainer />
```

Comment:
Voor nu is dit oke, ik denk dat we in de toekomst hier wel een page van willen maken. (Working on the routing)